lib/cisco_node_utils/cmd_ref/span_session.yaml

```diff
@@ -0,0 +1,65 @@
+# SPAN session
+---
+_template:
```

Comment:
I made some edits to this file to address the following (diffs below)
* Consolidate `get_context` and `set_context` under `_template` for all applicable properties
* Fixed `destination` `get_value` regexp.
* Fixed `type` `get_value` regexp.

```ruby
diff --git a/lib/cisco_node_utils/cmd_ref/span_session.yaml b/lib/cisco_node_utils/cmd_ref/span_session.yaml
index fd4d2b2..9e78b84 100644
--- a/lib/cisco_node_utils/cmd_ref/span_session.yaml
+++ b/lib/cisco_node_utils/cmd_ref/span_session.yaml
@@ -3,6 +3,8 @@
 _template:
   nexus:
     get_command: "show running monitor all"
+    get_context: ["monitor session <id>(?: .*)?"]
+    set_context: ["monitor session <id>"]
 
 all_sessions:
   multiple:
@@ -10,56 +12,52 @@ all_sessions:
   get_value: '/^monitor session (\d+)/'
 
 create:
+  set_context: ~
   set_value: 'monitor session <id>'
 
 description:
   kind: string
-  get_context: ["monitor session <id>"]
   get_value: '/^description (.*)/'
-  set_context: ["monitor session <id>"]
   set_value: '<state> description <description>'
   default_value: ''
 
 destination:
   kind: string
-  get_context: ["monitor session <id>"]
-  get_value: '/^destination interface <intf_name>$/'
-  set_context: ["monitor session <id>"]
+  get_value: '/^destination interface (\S+)$/'
   set_value: '<state> destination interface <intf_name>'
 
 destroy:
+  set_context: ~
   set_value: 'no monitor session <id>'
 
 session_id:
   kind: int
   default_value: false
+  get_context: ~
   get_value: '/^monitor session (\d+)/'
+  set_context: ~
   set_value: '<state> monitor session <id> <type> <val>'
 
 shutdown:
   kind: boolean
   default_value: true
-  get_context: ["monitor session <id>"]
   get_value: '/^no shut$/'
-  set_context: ["monitor session <id>"]
   set_value: '<state> shut'
 
 source_interface:
   kind: string
-  get_context: ["monitor session <id>"]
   get_value: '/^source interface (\S+) (\w+)$/'
-  set_context: ["monitor session <id>"]
   set_value: '<state> source interface <int_name> <direction>'
 
 source_vlan:
   kind: int
-  get_context: ["monitor session <id>"]
   get_value: '/^source vlan (\S+) (\w+)$/'
-  set_context: ["monitor session <id>"]
   set_value: '<state> source vlan <vlans> <direction>'
 
 type:
   kind: string
   default_value: 'local'
-  get_value: '/^monitor session \d+ type (\w+)$/'
+  get_context: ~
+  get_value: '/^monitor session \d+ type (\S+)$/'
+  set_context: ~
   set_value: 'monitor session <id> type <type>'
```